pkg/console/operator/sync_v400.go

```diff
@@ -200,12 +200,13 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 func (co *consoleOperator) SyncConsoleConfig(consoleConfig *configv1.Console, consoleURL string) (*configv1.Console, error) {
 	oldURL := consoleConfig.Status.ConsoleURL
 	updated := consoleConfig.DeepCopy()
+	metrics.HandleConsoleURL(oldURL, consoleURL)
```

Comment:
Do we need to cal this when the URL hasn't changed?

Reply:
> Do we need to cal this when the URL hasn't changed?

`HandleConsoleURL` is handling all the cases, even if the URL has not changed, then it will just return.